ci-operator/config/openshift/ironic-inspector-image/openshift-ironic-inspector-image-release-4.8__prevalidation.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  ocp_builder_rhel-8-base-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-base-openshift-4.8
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
+  to: ironic-inspector
+raw_steps:
+- pipeline_image_cache_step:
+    commands: |
+      cat <<EOF> /etc/yum.repos.d/base-4-8-rhel-8-ironic-prevalidation.repo
+      [rhel-8-ironic-prevalidation-ocp48]
+      id = rhel-8-ironic-prevalidation
+      name = rhel-8-ironic-prevalidation
+      baseurl = http://base-4-8-rhel-8-ironic-prevalidation.ocp.svc/rhel-8-ironic-prevalidation-ocp48
+      enabled = 1
+      gpgcheck = 0
+      priority = 1
+      skip_if_unavailable=FALSE
+      EOF
+    from: ocp_builder_rhel-8-base-openshift-4.8
+    to: base
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.8"
+  namespace: ocp
+tests:
+- as: e2e-metal-ipi-prevalidation
+  steps:
+    cluster_profile: packet
+    workflow: baremetalds-e2e
```

Comment:
I'd recommend to always define explicitly the `DEVSCRIPTS_CONFIG` here to avoid unwanted issues, ie:

```
    env:
      DEVSCRIPTS_CONFIG: |
        IP_STACK=v4
        NETWORK_TYPE=OpenShiftSDN
```

Reply:
oh right, thanks for the reminder!